templates/zerver/help/see-if-user-here-away.md

```diff
@@ -0,0 +1,35 @@
+### User List
+
+You can see if a user is active on Zulip by looking at the user list located on the far right side of your screen.
+
+![User List](/static/images/help/user-list.png)
+
+The **Search people** field at the top of the user list allows you to search for a specific user. As you type the user's name, the user list will shrink to show only people with that name.
+
+![Search Field](/static/images/help/search-field-user-list.png)
+
+If you mouse over a user's name on the user list, you will see a small chevron (![chevron](/static/images/help/chevron.png)).
```

Comment:
I'm not sure this is relevant in the context of this doc - the chevron here is used for narrowing to or composing private messages. Was it mentioned in a Zulip discussion?